WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutPickerViewModel.kt

```diff
@@ -65,6 +65,8 @@ abstract class LayoutPickerViewModel(
 
     abstract val useCachedData: Boolean
 
+    open var nestedScrollStates: Bundle = Bundle()
```

Comment:
I think we can omit the `open` keyword

Reply:
Having the `nestedScrollStates` in the ViewModel has the minor side effect that it persists the horizontal scroll positions when the layout chooser closes and reopens. I think it might work better if we instead moved the `nestedScrollStates` in the [LayoutPickerUiState](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/layoutpicker/LayoutPickerUiState.kt) or in the `HomePagePickerFragment` and `ModalLayoutPickerFragment`. I haven't tried that though thus consider this just a suggestion.

Reply:
removed open keyword
Looked into your suggestion for moving scrollState into some UiState, it is not available for access as it is nested through viewholders and recyclerviews.  It would have been easier if they were all just recyclerviews without nesting inside viewholders I guess.